### ISSUE TITLE
Open widget's panel close to the click position

### ISSUE THREAD

**Issue by whimboo** [mozilla/memchaser#113]
As long as we can't use a real context menu the widget element will open a panel when clicked with some additional features. See issue #88. This panel is not positioned at the click position right now and can't even not be done from the extension. 

This issue is blocked by: https://bugzilla.mozilla.org/show_bug.cgi?id=741371

**Comment by xabolcs** [comment]
Wontfix, isn't it?

**Comment by whimboo** [comment]
The bug got wontfix and current behavior seem to be ideal.
